lecture/views.py

```
@@ -31,19 +67,46 @@ def lecture_home(request: HttpRequest) -> HttpResponse:
 
     return render(request, 'lecture/index.html', {
         'course': course,
-        'lectures': lectures
+        'lectures': lectures,
+        'type':"pdf"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)